jsdrupal/drupal-admin-ui
# Fetch Schema Automatically

## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/221

## Screenshot / UI changes

No UI changes, just fetching the schema automatically.

## Testing instructions

Ensure that you have the latest support module enabled, and openapi/schemata. Ensure that `/node/add/recipe` works as expected.
